SKNIBot/Core/Database/Models/DynamicDB/UserMessageStats.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace SKNIBot.Core.Database.Models.DynamicDB
+{
+    public class UserMessageStat
+    {
+        public UserMessageStat()
+        {
+            count = 0;
+            ChannelID = 0;
+            User = 0;
+        }
+        public UserMessageStat(int cnt, ulong channel, ulong user)
+         {
+             count = cnt;
+             ChannelID = channel;
+             User = user;
+         }
+        public int ID { get; set; }
+        public int ServerID { get; set; }
+        public ulong ChannelID { get; set; }
```

Comment:
In SQLite Integer can use 8 bytes, but it's signed integer. Are you sure putting there ulong won't overflow it causing issues?
https://www.sqlite.org/datatype3.html
Previously we used string to keep ulong Discord ids.